popmon/visualization/utils.py

```diff
@@ -25,6 +25,8 @@
 
 import numpy as np
 import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
 import pybase64
 from matplotlib import pyplot as plt
```

Comment:
Already wrote it, but I think if we can switch fully to plotly, we probably should just do that.